README.md

```
@@ -65,16 +69,16 @@ That's it. `analyze` indexes the codebase, installs agent skills, registers Clau
 >
 > ```bash
-> pnpm --allow-build=@ladybugdb/core --allow-build=gitnexus --allow-build=tree-sitter dlx gitnexus@latest analyze
+> gitnexus analyze
 > ```
 >
-> Or install globally (`npm install -g gitnexus@latest`) and run `gitnexus analyze`. See [#1939](https://github.com/abhigyanpatwari/GitNexus/issues/1939).
+> The Electric fork is installed from an exact GitHub release tarball; it does not publish npm dist-tags.
 
-> **Fastest MCP startup:** install globally (`npm i -g gitnexus`) before running `gitnexus setup` — this writes an absolute-path MCP config that bypasses `npx` entirely. On a cold cache, an `npx`-based MCP install can exceed Claude Code's `MCP_TIMEOUT` default (~30s).
+> **Fastest MCP startup:** install the exact Electric GitHub release tarball before running `gitnexus setup` — this writes an absolute-path MCP config that bypasses package-manager startup.
 
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)